test_data/integration/null_safety/src/null_safety.dart

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2021, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// @dart=2.8
```

Comment:
I'd recommend testing boundary conditions, so consider using `2.11` instead. We should also have a test with `2.12` to ensure that we're not generating a lint in that case.